spec/controllers/api/v1/ontology_versions_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
   context 'on GET to index' do
     context 'requesting json representation', api_specification: true do
       let(:ontology_versions_schema) do
-        schema_for('ontology/commands/ontology_versions')
+        schema_for_command('ontology/ontology_versions', :get, 200)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.